libc/benchmarks/gpu/LibcGpuBenchmark.cpp

```diff
@@ -92,23 +92,22 @@ BenchmarkResult benchmark(const BenchmarkOptions &options,
   for (int i = 0; i < overhead_iterations; i++)
     overhead = cpp::min(overhead, LIBC_NAMESPACE::overhead());
 
-  for (uint64_t time_budget = options.max_duration; time_budget >= 0;) {
+  for (uint64_t time_budget = options.max_duration; time_budget != 0;) {
```

Comment:
This should definitely be `int64_t`. There's a *very* slim chance you will actually get zero if you keep subtracting.  Remember, integer overflow / underflow is not undefined behavior for unsigned values, so `1 - 2 == 18,446,744,073,709,551,615`.

Reply:
Oops, I totally missed this